### PR TITLE
Update readme file to include installation instructions in the case o…

### PR DESCRIPTION
**Pull request by lmazzoli**
Closes Issue #28

Added comments for windows and MacOS to configure Python to use the system certificate store.

DCO 1.1 Signed-off-by: Leo Mazzoli lmazzoli@us.ibm.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ All of the watson-stt-wer-python dependencies are installed at once with `pip`:
...
+**Note:**  If receiving an SSL Certificate error (CERTIFICATE_VERIFY_FAILED) when running the python scripts, try the following commands to tell python to use the system certificate store.
+
+**_Windows_**
+```
+pip install --trusted-host pypi.org --trustedhost files.python.org python-certifi-win32
+```
+
+**_MacOS_**
+
+Open a terminal Windows and change to the location of your python installation
```
Comment (andrewrfreed) [inline README.md:27]: Suggestion:
Open a terminal and change to the location of your python installation to execute `Install Certificates.command`, for example:

**Review by andrewrfreed — CHANGES_REQUESTED** [review]
Minor wording improvement suggested

**Review by gus07ven — COMMENTED** [review]
I like the wording of Andrew's suggestion. Everything else looks good Leo.

File: README.md (hunk 1 above)
Comment (lmazzoli) [reply README.md]: @andrewrfreed @gus07ven Good suggestion.  Update has been made.

**Review by gus07ven — APPROVED** [review]

**Review by andrewrfreed — APPROVED** [review]

**Pull request merged**